test/performance/observed-concurrency/reporter.lua

```diff
@@ -14,7 +14,16 @@
 
 -- parses a response as returned by the system under test
 function parse_response(res)
+  if (res == nil or res == '') then
+    print('ERROR: Test produced no response.')
+    os.exit()
+  end
   local split_at, _ = res:find(',', 0)
+  if (split_at == nil or split_at == '') then
```

Comment:
I suggest "ERROR: Empty response from test", it's clearer.

Reply:
updated